src/applications/appeals/shared/components/ContestableIssues.jsx

```diff
@@ -203,7 +206,8 @@ const ContestableIssues = props => {
   return (
     <>
       <div name="eligibleScrollElement" />
-      {showNoIssues && <NoIssuesLoadedAlert />}
+      {showApiFailure && <ApiFailureAlert />}
```

Comment:
Does this work like an if else statement where if the first condition isn't met, it checks the next boolean?

Reply:
In this pattern, if the first value is truthy, the component after the `&&` is rendered; if the value is falsy, the line is ignored.